Neos.Flow/Classes/Cache/CacheFactory.php

```diff
@@ -57,18 +57,20 @@ class CacheFactory extends \Neos\Cache\CacheFactory
 
     /**
      * @param CacheManager $cacheManager
-     * @Flow\Autowiring(enabled=false)
+     *
+     * @Flow\Autowiring (enabled=false)
```

Comment:
Why? I could live with the empty line, but the space before the parentheses looks very wrong!

Reply:
I'll fix that in a separate PR.